### PR TITLE
[CPM-153] Add more programmes to plan creation

### PR DESCRIPTION
**Pull request by daltonlim**
## Changes

- User can now select a programme to create a plan for
- Plan page now has a name field

## Checklist

- [x] Have tests been written?
- [x] Is the UI consistent?

## Requests
- [ ] Can somebody check the styling/CSS?

Addresses [CPM-153]

Connects #y

[CPM-153]: https://momentum-agile.atlassian.net/browse/CPM-153

### CODE REVIEW

**Comment by FocalChord** [comment]
This button looks a bit squished, could you change it pls

Screen Shot 2020-09-24 at 2 39 57 PM

File: client/src/pages/Students/ViewStudent.js [hunk 1]
```diff
@@ -104,17 +107,36 @@ const ViewStudent = ({ student, editStudent, deleteStudent }) => {
...
+                            backgroundColor={c.uoaBlue}
+                            marginLeft="20px"
+                            isDisabled={!programmeID}
+                            onClick={() =>
+                                CoursePlannerClient.createStudentPlan(editUpi, {
+                                    name: `${name}'s ${programmes.find((programme) => programme._id === programmeID).name} plan`,
+                                    student: student._id,
+                                    programmeDegree: programmeID,
+                                    startYear: new Date().getFullYear() + 1,
+                                    endYear: new Date().getFullYear() + 1,
```
Comment (FocalChord) [inline client/src/pages/Students/ViewStudent.js:131]: Should Nasser be able to define these 0.o ?

Comment (FocalChord) [reply client/src/pages/Students/ViewStudent.js:131]: Instead of being hardcoded

Comment (daltonlim) [reply client/src/pages/Students/ViewStudent.js:131]: Is that covered by the edit plan

Comment (FocalChord) [reply client/src/pages/Students/ViewStudent.js:133]: No it is not at the moment

Comment (daltonlim) [reply client/src/pages/Students/ViewStudent.js:133]: I feel like it should default to current year and changes can be made on the plan page. Similar to how you create a new document on drive but edit the title afterwards.

Comment (FocalChord) [reply client/src/pages/Students/ViewStudent.js:133]: Okay that sounds good

**Review by FocalChord — APPROVED** [review]
Looks good!

**Review by Richardng12 — APPROVED** [review]
lgtm

**Pull request merged**